custom_components/zaptec/api.py

```diff
@@ -327,44 +306,374 @@ async def cancel_stream(self):
                 # this will still raise a exception, I think its a 3.7 issue.
                 # recheck this when the i have updated to 3.9
 
-            self._stream_task = None
+            finally:
+                self._stream_task = None
+
+    #-----------------
+    # API methods
+    #-----------------
+
+    async def installation_info(self) -> TDict:
+        '''Raw request for installation data'''
+
+        # Get the installation data
+        data = await self._account._request(f"installation/{self.id}")
```

Comment:
Same for this one? Why not use the api?

Reply:
There is currently no method in `class Account` for this as the only consumer of this request is in the `class Interface`.